include/net/lwm2m.h

```diff
@@ -79,6 +79,7 @@ struct lwm2m_ctx {
 	struct coap_reply replies[CONFIG_LWM2M_ENGINE_MAX_REPLIES];
 	struct k_delayed_work retransmit_work;
 	struct sys_mutex send_lock;
+	void *processed_req;
```

Comment:
Can't this be `struct lwm2m_message`?

Reply:
`struct lwm2m_message` is declared in a private LwM2M header (https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/net/lib/lwm2m/lwm2m_object.h#L411) and has dependencies to some other structures declared in there. Using it here would require to make it available in the public header, along with all these other structures it consists of and I'd rather avoid that.

Reply:
Hmm, ok. In that case could you add comment there why it is void and what is the correct type, now it is not obvious why we do not have proper type there.

Reply:
Ok, I can do that.